### PR TITLE
Cmis

### PR DESCRIPTION
**Pull request by bwrsandman**
Port of https://code.launchpad.net/~savoirfairelinux-openerp/knowledge-addons/cmis/+merge/212258

re-port of https://github.com/OCA/knowledge/pull/6

This module allows to configure a CMIS backend in OpenERP.
## LP reviews:
- @hbrunn: Approve on 2014-04-07 
- @max3903: Needs Fixing on 2014-04-24
  
  > @ehdem
  > l205: Can you put CMIS versions (1.0 or 1.1, not sure) ? 1.7 is a Magento version.
  > Thanks
- @bwrsandman: Needs Information on 2014-05-14 
  
  > Looking a lot better, still issues about the comments preceding functions which would be more useful as docstrings.

### CODE REVIEW

**Comment by max3903** [comment]
:+1:

File: cmis/cmis_backend_view.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                <group colspan="4" col="4">
+                  <field name="browsing_ok"/>
+                  <newline/>
+                  <field name="initial_directory_read" attrs="{'invisible':[('browsing_ok','=',False)]}"/>
+                  <button name="check_directory_of_read"
+                          type="object"
+                          class="oe_highlight"
+                          attrs="{'invisible':[('browsing_ok','=',False)]}"
+                          string="Test Directory"/>
+                  <!--<group attrs="{'invisible':[('browsing_ok','=',False)]}"
```
Comment (veloutin) [inline cmis/cmis_backend_view.xml]: Dead XML, this should be removed.

Comment (bwrsandman) [inline cmis/cmis_backend_view.xml]: Done. However this is not my code.

**Comment by veloutin** [comment]
:+1:

**Comment by JehoG** [comment]
:+1:

**Pull request merged**